package.json

```diff
@@ -20,10 +20,13 @@
   "license": "ISC",
   "description": "",
   "dependencies": {
-    "@applitools/eyes-cypress": "^3.10.3",
-    "cypress": "^4.5.0",
+    "@applitools/eyes-cypress": "^3.22.0",
+    "cypress": "^7.5.0",
```

Comment:
I had to upgrade Cypress due to [this bug](https://github.com/cypress-io/cypress/issues/7915).
This is a breaking change since updating dependencies to the latest versions requires users to have Node.js 12 (the required version before the change was Node.js 8).

I suggest releasing the plugin under a major version change and [reading this](https://github.com/netlify/plugins/blob/main/docs/CONTRIBUTING.md#major-releases).

Reply:
when you say a "major" version, i would expect that to be [major].[minor].[patch], but it seems that "major" may be different in this context, as i see some "major" versions in the netlify plugins list as patch updates. are you just referring to any large breaking update? just wanted to clarify before submitting that

Reply:
Hi @colbyfayock, sorry for the delay, not sure how I missed this comment. I meant a major semver version, e.g. this should have been released as `2.0.0` since it's a breaking change for **_this package_**, unrelated to the versioning in our plugins directory.
That's the reason [this PR](https://github.com/netlify/plugins/pull/359) is failing.

I think you could `npm deprecate` the released versions (all versions `>=1.4.0`) and release `2.0.0`, which should sort this out.

Reply:
thanks learning a lot from this :) deprecated 1.5.0 https://www.npmjs.com/package/netlify-plugin-visual-diff/v/1.5.0